Essentials/src/com/earth2me/essentials/Settings.java

```diff
@@ -1414,4 +1411,9 @@ public boolean isAddingSuffixInPlayerlist() {
     public int getNotifyPlayerOfMailCooldown() {
         return config.getInt("notify-player-of-mail-cooldown", 0);
     }
+
+    @Override
+    public long getMotdDelay() {
+        return config.getInt("delay-motd", 0) / 50;
```

Comment:
I would not divide by 50 here. Instead, I'd do it in the usage itself. That way you can potentially access the true value of it and use it elsewhere.